section/results.tex

```diff
@@ -12,15 +12,25 @@ \subsection{Evaluation Against Other Approaches}
     A ratio greater than $1$ indicates a slower execution time (\textbf{lower is better}).
     The shape index approach performs similarly or better than the other methods, except in the case of S4.
     }
+    \rv{Why the ratio? I find this a very weird visualization. It's not a competition. Show the performance of each of the techniques, in absolute values. Resort to logarithmic or interrupted bars (\url{https://tex.stackexchange.com/questions/83138/how-to-draw-interrupted-bar-charts-in-pgfplots}) if needed.}
+    \rv{Also I learn nothing from all of the flat orange lines; can't even see it.}
     \label{fig:compApproach}
 \end{figure*}
 
 \subsubsection{Analysis of the Query Execution Time}
-Figure~\ref{fig:compApproach} shows that the shape index approach performs better or comparably to the state-of-the-art Solid Pod network traversal algorithms, for all query templates except S4.
+Figure~\ref{fig:compApproach} shows that the shape index approach performs better or comparably
+\rv{be specific; what is \enquote{better}, what is \enquote{comparably}. Use exact words, and numbers where applicable.}
+to the state-of-the-art Solid Pod network traversal algorithms, for all query templates except S4.
 Figure~\ref{fig:compApproachRaw} in the \hyperref[sec:appendix]{Appendix} presents the execution times of each method and Table~\ref{tab:statSignificanceStateOfTheArt} presents the statistical significance by query templates.
-The shape index approach allows for answering queries from the S7 template, which was not possible by the other approaches, due to large amount of HTTP requests resulting in timeouts.
+The shape index approach allows for answering queries from the S7 template,
+\rv{Be specific and scientific here. It's not \enquote{answering}, it's obtaining results. The other approaches were \emph{able} to process the query, but not to produce the (non-zero) result~set within the timeout.}
+which was not possible by the other approaches, due to large number of HTTP requests resulting in timeouts.
+\rv{More specific, again: because a~large number of requests would be needed, and could not be completed within the chosen timeout.}
 Queries can require as little as 13\% of the execution time (S1 being 7 times faster) of the type index.
-The queries that perform the best are those in which the number of HTTP requests decreased the most, as can be inferred from the analysis of Table~\ref{tab:ratioUsefulResources}.
+\rv{Be more exact/precise. The \enquote{execution time of the type index} is not a thing.}
+The queries that perform the best are those in which the number of HTTP requests decreased the most,
+\rv{That we already knew before we started, what's new here?}
```

Comment:
I am not sure, I think we know it if the reduction of execution time due to the reduction of HTTP request is lesser than the overhead of the approach. But it is the goal of the paper to find that.